.github/test-piv.sh

```diff
@@ -45,18 +45,36 @@ popd
 java -noverify -cp PivApplet/bin/:jcardsim/target/jcardsim-3.0.5-SNAPSHOT.jar com.licel.jcardsim.remote.VSmartCard PivApplet/test/jcardsim.cfg >/dev/null &
 PID=$!
 sleep 5
+
+export OPENSC_CONF="${PWD}/.github/opensc-piv.conf"
```

Comment:
nit: I would put here a comment why we do this exercise (need to force avoid compat/bugs flags enforced in the PIV driver for this very generic ATR).

Or just write it as a comment to the conf file.

Reply:
I think the comment here should say that we want the card to be detected as contactless, so that we can test the access policies, right? I'd like to see SM tests as well, but I don't think there is any simulation that supports SM, or is there?

Reply:
I don't think it has anything to do with contact vs contactless. Had to do with card-piv.c matched ATR to a PIVKey card that used the same generic ATR and this set some restrictions on EC keys. 

PIV cards should enforce on card access rights over contactless. (Some Yubikey cards don't so card software does not enforce them unless card appears to be gov issued card.)  

No simulation that supports SM, that I know about. Part of the reason to get PIV SM into OpenSC was so applet developers have some way to test their SM.